cmd/notary/integration_test.go

```diff
@@ -838,21 +838,6 @@ func TestClientDelegationsPublishing(t *testing.T) {
 	_, err = runCommand(t, tempDir, "-s", server.URL, "publish", "gun")
 	require.NoError(t, err)
 
-	// Now remove this key, and make a new file to import the delegation's key from
-	require.NoError(t, os.Remove(filepath.Join(keyDir, canonicalKeyID+".key")))
-	tempPrivFile, err := ioutil.TempFile("/tmp", "privfile")
-	require.NoError(t, err)
-	defer os.Remove(tempPrivFile.Name())
-
-	// Write the private key to a file so we can import it
-	_, err = tempPrivFile.Write(privKeyBytesNoRole)
-	require.NoError(t, err)
-	tempPrivFile.Close()
-
-	// Import the private key, associating it with our delegation role
-	_, err = runCommand(t, tempDir, "key", "import", tempPrivFile.Name(), "--role", "targets/releases")
```

Comment:
just leaving a breadcrumb comment to make sure we test the equivalent flow of importing a delegation key for delegation roles to publish in part 3 -- excited to see it all come together :)